.github/workflows/ci.yml

```diff
@@ -250,10 +255,10 @@ jobs:
         run: |
           apt-get update
           apt-get install -y git openssh-client
-          
+
```

Comment:
I didn't notice that I also committed these. Whitespace changes should ideally be separate out (into its own commit) from the rest of  the functional changes.

Reply:
Think it's okay as it is.